integration/presto/src/test/scala/org/apache/carbondata/presto/integrationtest/PrestoInsertIntoTableTestCase.scala

```diff
@@ -188,8 +188,22 @@ class PrestoInsertIntoTableTestCase
     val absoluteTableIdentifier: AbsoluteTableIdentifier = getAbsoluteIdentifier("testdb",
       "testtable")
     val carbonTable = SchemaReader.readCarbonTableFromStore(absoluteTableIdentifier)
-    val segmentFoldersLocation = CarbonTablePath.getPartitionDir(carbonTable.getTablePath)
-    assert(FileFactory.getCarbonFile(segmentFoldersLocation).listFiles(false).size() == 8)
+    val segment0 = CarbonTablePath
+      .getSegmentPath(carbonTable.getAbsoluteTableIdentifier.getTablePath, "0")
```

Comment:
please add a method and send the segment number and then add assert inside that method

Reply:
done